styles/globals.css

```diff
@@ -148,3 +148,6 @@ div.react-terminal-active-input {
   outline: #243142;
   border-radius: 4px;
 }
+.notfound-img{
+  background-image: url("/public/assets/images/404.jpg");
+ }
```

Comment:
because we are using Next/Image we shouldn't need this in our global.css

Reply:
@sabakhilji Just want to follow up with this! after this code blcok is removed this PR should be RFM!

Reply:
oh, thanks! I completely forgot this. Done now @benalleng